src/sections/Workshop-grid/index.js

```diff
@@ -106,10 +107,25 @@ const WorkshopsPage = ({hide_path}) => {
                                                     </a> : ""}
                                             </div>
                                             {frontmatter.status === "delivered" ? "" : <p>Upcoming...</p>}
-                                            <button type="button" className="readme-btn" onClick={() => {
-                                                setID(id);
-                                                setContent(!content);
-                                            }}>
+                                            <button type="button" className="readme-btn" onClick={open ? 
+                                                ID === id ? 
```

Comment:
Create a separate function out of this so that that it's easier to read.

Reply:
I tried that But I couldn't. Can you direct me to how it can be done? @Tanuj22

Reply:
Basically, pass `id` as an argument to the function. Something like this `onClick={()=>handleClick(id)}` (might we wrong about the exact syntax) and in the definition of the function use normal if else for doing the necessary. If you need further help ping me on slack.